autogen/agentchat/conversable_agent.py

```diff
@@ -597,7 +597,7 @@ def receive(
         if request_reply is False or request_reply is None and self.reply_at_receive[sender] is False:
             return
         reply = self.generate_reply(messages=self.chat_messages[sender], sender=sender)
-        if reply is not None:
+        if reply is not None and reply:
```

Comment:
This will break things. Why?

Reply:
It is actually not needed. I removed it.